libs/wingsdk/src/target-tf-aws/topic.ts

```diff
@@ -19,7 +20,9 @@ export class Topic extends cloud.TopicBase {
   constructor(scope: Construct, id: string, props: cloud.TopicProps = {}) {
     super(scope, id, props);
 
-    this.topic = new SnsTopic(this, "Default", {});
+    this.topic = new SnsTopic(this, "Default", {
+      name: this.sanitizeName(`${this.node.id}-${this.node.addr}`),
```

Comment:
I'd like to be able to set name via `props`

Reply:
Same before

`new cloud.Topic() as "MyAmazingTopic";`

AWS Console -> `MyAmazingTopic-<addr>`